### PR TITLE
Make it easier to run webhook tests locally

### PR DESCRIPTION
**Pull request by rbren**
You should now be able to use the included instructions to test the webhook locally.

### CODE REVIEW

File: docs-md/contributing.md [hunk 1]
```diff
@@ -32,12 +32,25 @@ We label issues with the ["good first issue" tag](https://github.com/FairwindsOp
...
 go list ./... | grep -v vendor | xargs golint -set_exit_status
 go list ./... | grep -v vendor | xargs go vet
 go test ./pkg/... -v -coverprofile cover.out
 ```
 
+### Webhook tests
+```bash
+kind create cluster --wait=90s --image kindest/node:v1.15.11 --name polaris-test
+docker build -t quay.io/fairwinds/polaris:debug .
+docker push quay.io/fairwinds/polaris:debug .
```
Comment (jordandoig) [inline docs-md/contributing.md:45]: I'm not sure the `.` on the end is necessary, I get `"docker push" requires exactly 1 argument.` when I run this

File: docs-md/contributing.md [hunk 2]
```diff
@@ -32,12 +32,25 @@ We label issues with the ["good first issue" tag](https://github.com/FairwindsOp
...
+helm repo add jetstack https://charts.jetstack.io
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version 0.16.1 --set "installCRDs=true" --wait
```
Comment (jordandoig) [inline docs-md/contributing.md:47]: Mind adding `kubectl create ns cert-manager` above this line as well?

**Review by jordandoig — APPROVED** [review]
Looks good to me, thanks for all the edits to make the tests run on my lowly Mac. I left a few comments around edits to the instructions, but otherwise should be set!

**Review by baderbuddy — APPROVED** [review]
LGTM

**Pull request merged**